base/server/src/main/java/com/netscape/cmscore/dbs/CertificateRepository.java

```diff
@@ -83,6 +83,7 @@
 
     private boolean mConsistencyCheck = false;
 
+    protected boolean mEnableRandomSerialNumbers;
```

Comment:
This can now be `private` if we're in the business of encapsulation.

Reply:
I think anything other than `public` would be considered encapsulation too, but it doesn't really matter since this class doesn't have any subclass that would be affected by `protected` vs `private`. I've updated the code regardless.